app-misc/brewtarget/brewtarget-4.0.15.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# All tests currently timeout even with a large timeout multiliplier
+# so they are commented out and disabled for now
+
+EAPI=8
+
+PLOCALES="ast ca cs da de el en en_GB es et eu fr gl hr hu it ja lv nb nl pl pt pt_BR pt_PT ru sr sv tr zh"
+#inherit meson plocale xdg-utils virtualx
+inherit meson plocale xdg-utils
+
+DESCRIPTION="Application to create and manage beer recipes"
+HOMEPAGE="https://www.brewtarget.beer/"
+SRC_URI="https://github.com/Brewtarget/brewtarget/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3 WTFPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+
+#RESTRICT="!test? ( test )"
+RESTRICT="test"
```

Comment:
RESTRICT may take more arguments, so no need to disable the above line, just both them all in one RESTRICT.

Reply:
I couldn't get the tests to work, they always timeout. So I just decided to disable them.